app/controllers/dashboards_controller.rb

```diff
@@ -11,6 +11,7 @@ def search
     end
     @patient = Patient.new
     @pregnancy = @patient.pregnancies.new
+    @date=Time.now.to_date
```

Comment:
@Nico-ehs what do you think about renaming this variable to something like `@today`? It's not really a generic date but today's date, so I think that might be a little easier to read on the fly. What do you think?